src/sentry/static/sentry/app/components/charts/panelChart/legend.jsx

```diff
@@ -0,0 +1,81 @@
+import {Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import CircleIndicator from 'app/components/circleIndicator';
+import space from 'app/styles/space';
+import theme from 'app/utils/theme';
+
+const Legend = styled(
+  class extends React.Component {
+    static propTypes = {
+      series: PropTypes.arrayOf(
```

Comment:
Should this be `area` or `areas` instead of `series` since `lines` below is also one type of series?